src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs

```diff
@@ -311,6 +311,15 @@ public static void Address_Property_Success()
             Assert.Equal(ip1, ip2);
             Assert.Equal(ip1.GetHashCode(), ip2.GetHashCode());
         }
+
+        [Fact]
```

Comment:
This is going to fail when it runs on .NET Framework.  Search the repo for the SkipOnTargetFramework attribute usage, and apply a similar attribute here to skip it when running on netfx.

Reply:
@stephentoub addressed in latest commit

Reply:
@stephentoub Any chance of this patch eventually making it's way into the .NET Framework?

Reply:
> Any chance of this patch eventually making it's way into the .NET Framework?

Very doubtful.